packages/v4/src/content/design-guidelines/components/navigation/navigation.md

```diff
@@ -29,27 +29,77 @@ Vertical navigation is hierarchical global navigation that displays navigation o
 * You have secondary navigation items (even if you have less than five primary navigation items)
 * You expect your application to be used on mobile devices
 
-#### Variations
+### Verical navigation variants
 
-**Simple vertical navigation**
+#### Simple vertical navigation
 
 Use a simple, single-level vertical navigation when you only have one level of navigation to display.
 
 <img src="./img/single-level.png" alt="Example of simple vertical navigation" width="634"/>
 
-**Grouped navigation**
+#### Grouped navigation
 
 When you have a small amount of secondary navigation items, you can group your items and display them persistently beneath the primary navigation items.
 
 <img src="./img/two-level.png" alt="Example of grouped vertical navigation" width="634"/>
 
-**Expandable vertical navigation**
+#### Expandable two-level navigation
 
 When you have a large number of secondary navigation items, you can use an expandable navigation to collapse and expand options as needed.
 
 <img src="./img/expandable-nav.png" alt="Example of expandable vertical navigation" width="634"/>
 
+#### Expandable three-level navigation
 
+When you have three levels of navigation items, you can use a three-level expandable navigation to expose tertiary navigation items. Expansion works well when there are small numbers of items at the current level and the entire page hierarchy can be visualized at the same time.
+
+<img src="./img/expandable-nav-tertiary.png" alt="Example of expandable tertiary vertical navigation" width="634"/>
+
+#### Fly-out navigation
+
+Fly-out navigation exposes navigation items nested within a parent node in an overlay panel that appears to the right of the parent item on hover or click. Fly-out menus can be used to expose secondary or tertiary levels of navigation. While fly-out menus can be cascaded, this is not recommended as the mouse interaction required to operate cascading menus can be difficult for some users.
+
+<img src="./img/fly-out.png" alt="Example of fly-out navigation" width="917"/>
+
+The advantage of fly-out menus is that they allow a user to easily scan through menu items in a deep page hierarchy. But keep in mind that the page in view may not be exposed as a selected menu item at the top level. Therefore, when using fly-outs, we strongly recommend use of [breadcrumbs](/components/breadcrumb) to help users understand where they are currently working within the site hierarchy.
+
+Also, consider how likely it is that your application will be used on a mobile phone. Fly-outs are not mobile friendly and supporting unique  menu patterns on different platforms (mobile vs desktop).
```

Comment:
Should it say "are not mobile friendly **in** supporting", instead of "**and**"supporting?
Also seems to be an extra space between "unique" and "menu"